GraphQL.Net/GraphQLTypeBuilder.cs

```diff
@@ -129,6 +129,12 @@ public void AddAllFields()
                 _type.OwnFields.Add(CreateGenericField(prop));
         }
 
+        public void AddAllFields(Dictionary<string, string> aliasDictionary)
```

Comment:
Another option is to provide an `AddAllFields(Func<string, string> transform)` overload. The user could then pass a method that transforms the prop name to whatever the desired output is. The default implementation would of course be `s => s.ToCamelCase()`.